package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "creditcards-types",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Please revert this, PRs should not modify the version directly, I do that via `npm version`

Reply:
done